web/annotation_layer_builder.css

```diff
@@ -93,6 +93,64 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation label,
+.annotationLayer .radioButtonWidgetAnnotation label {
+  background-color: rgb(222, 229, 255);
+  border: 1px solid transparent;
+  box-sizing: border-box;
+  cursor: pointer;
+  height: 100%;
+  position: absolute;
+  width: 100%;
+}
+
+.annotationLayer .checkboxWidgetAnnotation input,
+.annotationLayer .radioButtonWidgetAnnotation input {
+  position: absolute;
+  left: -9999px;
+}
+
+.annotationLayer .radioButtonWidgetAnnotation label {
+  border-radius: 50%;
+}
+
+.annotationLayer .checkboxWidgetAnnotation label:hover,
+.annotationLayer .radioButtonWidgetAnnotation label:hover {
+  border: 1px solid #000;
+}
+
+.annotationLayer .checkboxWidgetAnnotation input:checked + label:before {
+  border-bottom: 1px solid #000;
+  border-left: 1px solid #000;
+  content: '';
+  height: 25%;
+  left: 50%;
+  position: absolute;
+  -webkit-transform: translate(-50%, -50%) rotateZ(-45deg);
+  -moz-transform: translate(-50%, -50%) rotateZ(-45deg);
+  -o-transform: translate(-50%, -50%) rotateZ(-45deg);
+  -ms-transform: translate(-50%, -50%) rotateZ(-45deg);
+  transform: translate(-50%, -50%) rotateZ(-45deg);
+  top: 50%;
+  width: 60%;
+}
+
+.annotationLayer .radioButtonWidgetAnnotation input:checked + label:before {
```

Comment:
There seems to be some overlap between these rules and the ones above, could that be simplified somewhat?